### ISSUE TITLE
Issue starting ACS Commons bulk workflow with smart tag workflow

### ISSUE THREAD

**Issue by badvision** [Adobe-Consulting-Services/acs-aem-commons#1276]
## Required Information

* [x] AEM Version, including Service Packs, Cumulative Fix Packs, etc: 6.4 L21
* [x] ACS AEM Commons Version: 3.11+
* [x] Reproducible on Latest? yes

### Expected Behavior

I would really like to run bulk workflow for smart tags.

### Actual Behavior

It acts like it's starting, creates an action manager (using multi-threaded mode with Query Builder) and it just dies.  Found this error in the log:

```07.03.2018 23:29:27.623 *ERROR* [192.150.9.200 [1520465367403] POST /etc/acs-commons/bulk-workflow-manager/smart-tags-for-celium/_jcr_content.start.json HTTP/1.1] com.adobe.granite.workflow.core.jcr.WorkflowModelManager Failed to check for the existance of a overlaid workflow model with ID /var/workflow/models//var/workflow/models/dam/dam-autotag-assets, continuing with original model ID /etc/workflow/models//var/workflow/models/dam/dam-autotag-assets/jcr:content/model
javax.jcr.RepositoryException: Invalid name or path: /var/workflow/models//var/workflow/models/dam/dam-autotag-assets
	at org.apache.jackrabbit.oak.jcr.session.SessionContext.getOakPathOrThrow(SessionContext.java:375) ```

**Comment by justinedelson** [comment]
is that the full stack trace?

**Comment by davidjgonzalez** [comment]
+1 to adding the full stack trace - took a look at the synth code and not clear where that would be happening (thought there are static references to the pre-6.4 /etc/workflow/models that needs adjusting; but are not this issue)

**Comment by badvision** [comment]
I'm pretty sure it's whatever converts that numeric value into a workflow path.  The UI doesn't have the workflow paths as selection values, so it's whatever happens on the other side of the POST of the setup part, I think.  Probably affects synthetic and plain workflow the same way because neither of them worked.

**Comment by davidjgonzalez** [comment]
Fixed via  #1306
